zulipterminal/cli/run.py

```diff
@@ -219,9 +208,25 @@ def main(options: Optional[List[str]]=None) -> None:
                            format(", ".join(complete))))
         print("   autohide setting '{}' specified {}."
               .format(*zterm['autohide']))
+        # For binary settings
+        # Specify setting in order True, False
+        valid_settings = {
+            'autohide': ['autohide', 'no_autohide'],
+        }
+        boolean_settings = dict()  # type: Dict[str, bool]
+        for setting, valid_values in valid_settings.items():
+            if zterm[setting][0] not in valid_values:
+                print("Invalid {} setting '{}' was specified {}."
+                      .format(setting, *zterm[setting]))
+                print("The following options are available:")
```

Comment:
Should we change 'options' to 'values'? Here or in the next commit?

Reply:
I think options is a more clear wording for it? ( + more user-friendly word)

Reply:
This isn't too important, I should have prefaced my comment with 'minor:' :)